mod/thewire/views/default/forms/thewire/add.php

```diff
@@ -33,28 +46,40 @@
 	$num_lines = 3;
 }
 
-$post_input = elgg_view('input/plaintext', [
+echo elgg_view_field([
+	'#type' => 'plaintext',
 	'name' => 'body',
 	'class' => 'mtm',
 	'id' => 'thewire-textarea',
 	'rows' => $num_lines,
 	'data-max-length' => $char_limit,
 	'required' => true,
-	'placeholder' => elgg_echo('thewire:form:body:placeholder'),
+	'placeholder' => $guid ? false : elgg_echo('thewire:form:body:placeholder'),
+	'value' => $guid ? $entity->description : '',
```

Comment:
same as above, are you sure `get_entity()` succeeded, check `($entity instanceof ElggWire)`